### PR TITLE
Add first iteration of list credential store

### PR DESCRIPTION
**Pull request by calcaide**
First iteration of list credential store. It is not complete is missing the action to create a new one, breadcrumbs and more styles.

Updated PR, now looks like:
Screen Shot 2021-07-12 at 7 34 07 AM

### CODE REVIEW

File: addons/core/translations/resources/en-us.yaml [hunk 1]
```diff
@@ -267,4 +267,7 @@ target:
       add:
         title: Add Host Sets
         description: Select host sets to assign to this target.
-    secret: ■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■
```
Comment (calcaide) [inline addons/core/translations/resources/en-us.yaml:270]: Cleaning up not used translations.

File: ui/admin/app/templates/scopes/scope/credential-stores.hbs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<Rose::Layout::Page as |page|>
+  {{! <page.breadcrumbs>
```
Comment (calcaide) [inline ui/admin/app/templates/scopes/scope/credential-stores.hbs:2]: Commented till I make them work properly (actually it does not show the proper breadcrumb)

File: ui/admin/app/templates/scopes/scope/credential-stores.hbs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  </page.breadcrumbs> }}
+
+  <page.header>
+    <h2>
+      {{t 'resources.credential-store.title_plural'}}
+    </h2>
+    {{t 'resources.credential-store.description'}}
+  </page.header>
+
+  {{!-- <page.actions>
```
Comment (calcaide) [inline ui/admin/app/templates/scopes/scope/credential-stores.hbs:13]: Commented till I create the route `new`

**Review by gsusmi — APPROVED** [review]
LGTM Ty!

**Pull request merged**